controllers/abstract_robot.py

```diff
@@ -0,0 +1,77 @@
+import struct
+from controller import Robot
+
+TIME_STEP = 64
+ROBOT_NAMES = ["B1", "B2", "B3", "Y1", "Y2", "Y3"]
+N_ROBOTS = len(ROBOT_NAMES)
+
+
+class AbstractRobot:
```

Comment:
@Adman maybe `RCJSimulatedRobot` or something like that may be a more appropriate name here? I really don't know though...

Reply:
Let's call it `RCJSoccerRobot` - since we mention `soccer` everywhere